cmd/run_test.go

```diff
@@ -50,5 +87,11 @@ func Test_runTestSuite(t *testing.T) {
 	}
 	want += "Testsuite completed in "
 	want = strings.Replace(want, "\n", "", -1)
+
+	// testing handleBlocks
+	assert.Equal(t, myTs.Blocks[0].Type, "init")
+	assert.Equal(t, myTs.Blocks[1].Type, "concurrent")
```

Comment:
does testing all three cases add anything to the test?

Reply:
was't sure whether to add the three cases as handleBlocks got covered from being called from runTestSuite, but thought the function needed to be specifically checked